README.md

```diff
@@ -78,3 +78,5 @@ The following instructions can be used to setup Dynamatic from source. If you in
     cd build
     ninja check-dynamatic
     ```
+
+    _**Important note:** If Dynamatic is built in Release mode, one of the tests will crash (see #164), hence it is recommended to run these tests in the Debug build._
```

Comment:
Please link the issue's URL on the "#164", so that it remains clickable even when the file is read outside a GitHub page.

Reply:
Fixed!